src/systemverilog/systemverilog.ts

```diff
@@ -393,21 +391,16 @@ export const language = <ILanguage>{
 		],
 
 		string: [
-			[/[^\\"]+/, 'string'],
-			[/@escapes/, 'string.escape'],
-			[/\\./, 'string.escape.invalid'],
-			[/"/, 'string', '@pop']
+			[/"/, 'string.escape', '@stringBody']
 		],
```

Comment:
I reviewed the Monaco docs and LRM and I think we'll want something almost the same as above.

The first line starts the match if the string is not within a comment
The second line highlights any escape sequences within the string (\n\t\r, \123, etc.).
- I think we'll need to update @escapes slightly - see this table in the LRM:
   - Table 21-1—Escape sequences for printing special characters
The 3rd line will highlight any invalid escape sequences not match by @escapes
The 4th line detects the end of the string and breaks out

Reply:
Which version of the LRM? In the 3.1 version I'm looking at table 21-1 has to do with parameter declaration.